embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/orchestrator/SessionOrchestratorImpl.kt

```diff
@@ -213,6 +213,12 @@ internal class SessionOrchestratorImpl(
                 }
                 Systrace.endSynchronous()
             }
+
+            if (activeSession == null && transitionType == TransitionType.ON_BACKGROUND) {
+                // if a new session hasn't been created when we background, cache an empty envelope to be used
+                // in case a native crash needs to be sent in the future after the current process dies
+                payloadStore?.cacheEmptyCrashEnvelope(payloadFactory.createEmptyLogEnvelope())
```

Comment:
How does the cache get cleaned up? Is that the responsibility of the resurrection service?